src/opensearch_sql_cli/formatter.py

```diff
@@ -1,97 +0,0 @@
-"""
-Copyright OpenSearch Contributors
-SPDX-License-Identifier: Apache-2.0
-"""
-
-import click
-import itertools
-
-from cli_helpers.tabular_output import TabularOutputFormatter
-from cli_helpers.tabular_output.preprocessors import format_numbers
-from opensearchpy.exceptions import OpenSearchException
-
-click.disable_unicode_literals_warning = True
-
-
-class Formatter:
```

Comment:
removing this breaks SQL doctest: https://github.com/opensearch-project/sql/blob/0af7429b2679279fe517e9d8c2a604e94f8cb27b/doctest/test_docs.py#L19-L21
```py
from opensearchsql_cli.formatter import Formatter
from opensearchsql_cli.opensearch_connection import OpenSearchConnection
from opensearchsql_cli.utils import OutputSettings
```

@Mebsina @Swiddis is there a new path that we should use? or are these no longer available?

Reply:
seems sql should stick with v1 https://github.com/opensearch-project/sql/pull/4292